### DISCUSSION TITLE
Can't delete stored procedure

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#1932] [Questions] [answered]
Hi,

I am trying to delete the stored procedure "ListOfWriters". It returns success when I run it, but it won't let me edit the procedure and says it still exists. I also shows up still in the side panel. Any ideas on why?
<img width="480" alt="Screen Shot 2021-06-11 at 12 21 38 PM" src="https://user-images.githubusercontent.com/47959617/121738984-dc5b7180-caaf-11eb-8577-7be21db53436.png">
<img width="398" alt="Screen Shot 2021-06-11 at 12 21 56 PM" src="https://user-images.githubusercontent.com/47959617/121739001-e0878f00-caaf-11eb-9da6-ae0a1bbef1c7.png">
<img width="262" alt="Screen Shot 2021-06-11 at 12 21 48 PM" src="https://user-images.githubusercontent.com/47959617/121739019-e5e4d980-caaf-11eb-959c-4beb923323c9.png">

**Comment by steve-chavez** [comment, marked as answer]
Try double quoting the function name:

```sql
drop function "ListOfWriters"(int);
```

Postgres is case insensitive if double quotes are not included.

**Reply by milestones95** [reply]
that worked! thanks
